config/secrets.yml

```diff
@@ -0,0 +1,22 @@
+# Be sure to restart your server when you modify this file.
+
+# Your secret key is used for verifying the integrity of signed cookies.
+# If you change this key, all old signed cookies will become invalid!
+
+# Make sure the secret is at least 30 characters and all random,
+# no regular words or you'll be exposed to dictionary attacks.
+# You can use `rake secret` to generate a secure secret key.
+
+# Make sure the secrets in this file are kept private
+# if you're sharing your code publicly.
+
+development:
+  secret_key_base: 436cc31267e1dfe7bffcf805e537eefb589cb47057a3e572ba9f46f71bd7b4b2bf6fe6653835ea3541aa2c0ba57b3de3d33b122e7a9923e8e6019cb7a9852a23
+
+test:
+  secret_key_base: 05842753351a9f1603a2e8a703c2c793c01143c216e5177b17bf8a937319926bc8a8ac2e4e828f0c60b07d8fac8bfe27e8dbfcb95e02ee229a9e976da85cbe22
```

Comment:
Do we want this file commited? It is usually a bad practice to commit keys even if they are just used for development. I can see that we are using similar approach in reg products but this file isn't present in that repo, not sure if that approach can be used here, just a suggestion.

Reply:
This is the default that happens when you create a new Rails project with `rails new`. I think the thinking is: projects are only run for real with `RAILS_ENV=production`, so the dev and test keys (which are randomly generated by every `rails new`) are benign to commit to source control. And if they weren't, every time we ran the test suite, locally or in CI, we'd need to capture the output of `rails secret` in an env var.

Tl;dr - good question to ask, but I think it's OK